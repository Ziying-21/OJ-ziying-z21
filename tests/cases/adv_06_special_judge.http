###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 99.0); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 373
# access-control-expose-headers: content-type
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# date: Fri, 09 Sep 2022 02:24:39 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:24:40.248Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 99.0); }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 449
# access-control-expose-headers: content-type
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# date: Fri, 09 Sep 2022 02:24:40 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":125771},{"id":1,"info":"The error is within bounds","memory":0,"result":"Accepted","time":291146}],"created_time":"2022-09-09T02:24:40.248Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 99.0); }","user_id":0},"updated_time":"2022-09-09T02:24:41.233Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 80.0); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 373
# access-control-expose-headers: content-type
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# date: Fri, 09 Sep 2022 02:24:40 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:24:41.262Z","id":1,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 80.0); }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/1 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 455
# access-control-expose-headers: content-type
# access-control-allow-credentials: true
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# date: Fri, 09 Sep 2022 02:24:42 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":122424},{"id":1,"info":"The error is beyond bounds","memory":0,"result":"Wrong Answer","time":329136}],"created_time":"2022-09-09T02:24:41.262Z","id":1,"result":"Wrong Answer","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() { println!(\"{}\", 80.0); }","user_id":0},"updated_time":"2022-09-09T02:24:41.740Z"}
