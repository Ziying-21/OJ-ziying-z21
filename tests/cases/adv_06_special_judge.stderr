[2022-09-09T02:24:39Z INFO  actix_server::builder] Starting 8 workers
[2022-09-09T02:24:39Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-09T02:24:40Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000175
[2022-09-09T02:24:40Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 373 "-" "-" 0.000549
[2022-09-09T02:24:41Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 449 "-" "-" 0.001540
[2022-09-09T02:24:41Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 373 "-" "-" 0.000948
[2022-09-09T02:24:42Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/1 HTTP/1.1" 200 455 "-" "-" 0.000346
